modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -439,9 +449,16 @@ impl ServerModule for Wallet {
 
         items.push(WalletConsensusItem::Feerate(fee_rate_proposal));
 
-        items.push(WalletConsensusItem::ModuleConsensusVersion(
-            MODULE_CONSENSUS_VERSION,
-        ));
+        if dbtx
+            .get_value(&ConsensusVersionVotingActivationKey)
+            .await
+            .is_some()
+            || is_running_in_test_env()
+        {
+            items.push(WalletConsensusItem::ModuleConsensusVersion(
+                MODULE_CONSENSUS_VERSION,
+            ));
+        }
```

Comment:
We should stop voting once consensus is achieved imo, otherwise we are just wasting consensus bandwidth.

Reply:
The bandwidth consumption is negligent. Optimizing this is would therefore be confusing as it suggests that its not.